deepchem/models/lightning/tests/test_dc_lightning_module.py

```diff
@@ -56,10 +56,61 @@ def __getitem__(self, index):
                                 dropouts=0.2,
                                 learning_rate=0.0001)
 
-    valid_dataloader = DataLoader(TestDataset(valid_dataset),
+    valid_dataloader = DataLoader(TestMultitaskDataset(valid_dataset),
                                   batch_size=64,
                                   collate_fn=collate_dataset_wrapper)
 
     lightning_module = DCLightningModule(model)
     trainer = pl.Trainer(max_epochs=1)
     trainer.fit(lightning_module, valid_dataloader)
+
+  @unittest.skipIf(PYTORCH_LIGHTNING_IMPORT_FAILED,
+                   'PyTorch Lightning is not installed')
+  def test_gcn_model(self):
+
+    class TestGCNDataset(torch.utils.data.Dataset):
```

Comment:
You don't need this class. Instead you can just use `make_pytorch_dataset()` called directly from the deepchem dataset

Reply:
hey @rbharath, one comment on this, Princy is working on a dataset module which will make this implementation easy to use with the `make_pytorch_dataset()` functionality from deepchem, my suggestion to Princy was to have this dataset as it is right now because she has setup the interaction with lightning module in a particular way, this will definitely be made simpler in another PR later this week.

Reply:
Sounds good!